cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -44,11 +44,25 @@ import (
 	utilflag "k8s.io/apiserver/pkg/util/flag"
 )
 
+// deprecatedOption can be used to define a deprecated command line
+// flag. Whenever it is set, regardless to which value, a warning is
+// triggered.
+type deprecatedOption struct {
```

Comment:
I think these could exist in a different `util` type package. Maybe a specific file for it